### DISCUSSION TITLE
Any news about usage with other js frameworks like Svelte or Solid or Vue?

### DISCUSSION THREAD

**Discussion by frederikhors** [orbitinghail/sqlsync#55] [Q&A] [answered]
Any news about usage with other js frameworks like Svelte or Solid or Vue?

**Comment by carlsverre** [comment, marked as answer]
There is a solidjs package here: https://github.com/orbitinghail/sqlsync/tree/main/lib/sqlsync-solid-js
I suspect it would be easy to follow that package's structure to create similar packages for Solid/Vue. Do you have a use case that needs one of those libraries?

**Reply by frederikhors** [reply]
Soon I will have to start a new project that will require synchronization between devices very often without internet.

I'm still not clear on how to make everything work with `sqlsync`, I'm still in the exploratory phase and the solidjs package certainly helps a lot.

As soon as I decide I can start writing something and why not also some guides/tutorials to make the work easier for other people.

Thanks for this wonder.
